app/relisten/(tabs)/artists/[artistUuid]/tours.tsx

```diff
@@ -1,16 +1,111 @@
-import { useArtistVenues } from '@/relisten/realm/models/venue_repo';
+import { useArtistTours } from '@/relisten/realm/models/tour_repo';
+import Flex from '@/relisten/components/flex';
+import { RefreshContextProvider } from '@/relisten/components/refresh_context';
+import { SubtitleRow, SubtitleText } from '@/relisten/components/row_subtitle';
+import RowTitle from '@/relisten/components/row_title';
+import { DisappearingHeaderScreen } from '@/relisten/components/screens/disappearing_title_screen';
+import { SectionedListItem } from '@/relisten/components/sectioned_list_item';
 import { useLocalSearchParams } from 'expo-router';
-import { Text, View } from 'react-native';
+import {
+  FilterableList,
+  FilterableListProps,
+} from '@/relisten/components/filtering/filterable_list';
+import { RelistenText } from '@/relisten/components/relisten_text';
+import Plur from '@/relisten/components/plur';
+import { FilteringProvider, Filter, SortDirection } from '@/relisten/components/filtering/filters';
+import { useHeaderHeight } from '@react-navigation/elements';
+import { Tour } from '@/relisten/realm/models/tour';
+import dayjs from 'dayjs';
 
 export default function Page() {
-  const { uuid } = useLocalSearchParams();
-
-  const results = useArtistVenues(String(uuid));
+  const { artistUuid } = useLocalSearchParams();
+  const results = useArtistTours(String(artistUuid));
   const { data } = results;
+  const headerHeight = useHeaderHeight();
+
+  return (
+    <RefreshContextProvider networkBackedResults={results}>
+      <DisappearingHeaderScreen
+        headerHeight={headerHeight}
+        ScrollableComponent={TourList}
+        tours={Array.from(data.tours)}
+        filterPersistenceKey={['artists', artistUuid, 'tours'].join('/')}
+      />
+    </RefreshContextProvider>
+  );
+}
+
+interface TourListItemProps {
+  tour: Tour;
+}
+
+const TourListItem = ({ tour }: TourListItemProps) => {
+  const startDate = dayjs(tour.startDate).format('MM/DD/YYYY');
+  const endDate = dayjs(tour.endDate).format('MM/DD/YYYY');
```

Comment:
we should come back and localize all date displays in the app to match the user's settings but we don't need to worry about that for now. if we aren't localizing we should put this in ISO format for now (`YYYY-MM-DD`)

Reply:
Makes sense. Updated.